addons/ModelViewer3D/documentation.md

```diff
@@ -1,6 +1,6 @@
 ## Description
 
-##### Model Viewer 3D allows for displaying 3D models. <br> Please note that when using a mobile device, an additional <img src="/file/serve/6093598547116032" style="border: 0; display:inline; margin: 0 2px; box-shadow: none" alt="AR"> icon will be visible in the vicinity of the 3D model. Interacting with the icon allows switching the preview to the Augmented Reality mode ([https://modelviewer.dev/examples/augmentedreality](https://modelviewer.dev/examples/augmentedreality "https://modelviewer.dev/examples/augmentedreality")).
+##### Model Viewer 3D allows for displaying 3D models. <br> Please note that when using a mobile device, an additional <img src="/file/serve/6093598547116032" style="border: 0; display:inline; margin: 0 2px; box-shadow: none" alt="AR"> icon will be visible in the vicinity of the 3D model. Interacting with the icon allows switching the preview to the Augmented Reality mode ([https://modelviewer.dev/examples/augmentedreality](https://modelviewer.dev/examples/augmentedreality "https://modelviewer.dev/examples/augmentedreality")) <br>Works on iPhones from iOS version 15.
```

Comment:
Czy dokumentacja jest poprawna ? W tickecie napisałeś, że działa od 14 jeżeli zastosuje się odpowiedni model. Tu piszesz, że od 15. 
Proszę popraw dokumentację

Reply:
może nie jest to dobrze wyjaśnione, ale chodzi o to że dla wersji iOS 15 działa poprawnie, natomiast dla modeli po konwersji z usdz na glb model-viewer działa od iOS 14

Reply:
Ok. Wydaje mi się, że i tak warto to opisać. Proszę dodaj w tickecie że masz propozycję, że w dokumentacji znajdzie się dopisek jak wyżej. I czy to jest wystarczające.